src/ui/steps/ShowAllLabelsStep.cpp

```diff
@@ -0,0 +1,20 @@
+//
+// Created by Anton Ovcharenko on 18.01.2022.
+//
+
+#include "ShowAllLabelsStep.h"
+#include "ui/Context.h"
+
+std::unique_ptr<Action> ShowAllLabelsStep::genAction(Context &context) {
+    auto labels = context.tasks()[0].data().labels();
```

Comment:
Unchecked access

Reply:
should not be a problem, but added a check nevertheless for good practice